### PR TITLE
created landing page

### LINKED ISSUE #6: Landing
src/comonents/landing
use wave1 and wave2 to created the design
you don't need to make the buttons but add the text
use AIS.svg for the logo 
Screenshot 2024-06-30 at 5 27 07 PM

### PR DESCRIPTION
**Pull request by zalam007**
image

### CODE REVIEW

**Comment by zalam007** [comment]
image

File: src/components/Landing.jsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
 import React from "react";
+import Image from "next/image";
+import wave1 from "../../public/wave1.svg";
+import wave2 from "../../public/wave2.svg";
+import AIS from "../../public/AIS.svg";
 
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="relative h-screen bg-blue-900 overflow-hidden">
```
Comment (stormyy00) [inline src/components/Landing.jsx]: do bg-gradient-to-b from- via- to-, pull from dev for the colors

File: src/components/Landing.jsx [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+      {/* Waves */}
+      <div className="absolute right-1/4 bottom-1/3 w-full h-full">
+        <Image src={wave1} alt="Wave 1" layout="fill" objectFit="cover" />
+      </div>
+      <div className="absolute right-1/4 bottom-1/4 w-full h-full">
+        <Image src={wave2} alt="Wave 2" layout="fill" objectFit="cover" />
+      </div>
+
+      {/* Content */}
+      <div className="absolute bottom-64 right-64 flex flex-col text-white">
```
Comment (stormyy00) [inline src/components/Landing.jsx]: use % so bottom-[20%]

File: src/components/Landing.jsx [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+        <h1 className="text-4xl mb-8">EMPOWERING FUTURE INNOVATORS</h1>
+
+        <div className="flex items-center mb-8">
```
Comment (stormyy00) [inline src/components/Landing.jsx]: try mb-4, looks to high up

**Review by stormyy00 — CHANGES_REQUESTED** [review]

**Comment by zalam007** [comment]
image

**Comment by zalam007** [comment]
Screenshot 2024-07-14 224401

**Review by stormyy00 — APPROVED** [review]

**Pull request merged**